src/app/app/onboarding/_components/OnboardingForm.tsx

```
@@ -3,140 +3,215 @@
   });
 
-  const nextStep: MouseEventHandler<HTMLButtonElement> = (e) => {
+  // Memoize avatar preview URLs and revoke on cleanup to prevent URL leaks
+  const avatarUrls = useMemo(
+    () => (avatarFiles ?? []).map((file) => URL.createObjectURL(file)),
+    [avatarFiles],
+  );
+
+  useEffect(() => {
+    return () => {
+      avatarUrls.forEach((url) => URL.revokeObjectURL(url));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)